core/server/common/src/main/java/alluxio/master/AbstractMaster.java

```diff
@@ -211,9 +149,9 @@ protected void writeJournalEntry(JournalEntry entry) {
    * Flushes the journal.
    */
   protected void flushJournal() {
-    Preconditions.checkNotNull(mJournalWriter, "Cannot flush journal: journal writer is null.");
+    Preconditions.checkNotNull(mJournal, "Cannot flush journal: journal is null.");
```

Comment:
Ideally include a descriptive error message. At minimum include the name of the null variable.